### DISCUSSION TITLE
✨ Introducing Metrics Insights!

### DISCUSSION THREAD

**Discussion by lowlighter** [lowlighter/metrics#229] [Announcements]
**v3.7** is introducing a new web feature named [Metrics Insights](https://metrics.lecoq.io/about)!

It is designed to create small web infographics that you can share with friends or share on social medias (just append your GitHub username at the end of the route, like [metrics.lecoq.io/about/lowlighter](https://metrics.lecoq.io/about/lowlighter). 

Here's what it looks like:

![preview](https://user-images.githubusercontent.com/22963968/114449143-b52e1400-9bd4-11eb-86b4-4db59ea549b4.png)

___

<details>

<summary>FAQ (click to expand)</summary>

**Q: Why?**
> Metrics is able to gather a huge amount of data and is designed to be highly configurable, but sometimes you just want to quickly check an user profile at a glance.
>
> With the recent improvements of **achievements** plugin which includes real leaderboard rankings, it seemed quite cool to be able to share these scores easily between friends or across social medias.
>
> Insights has been designed to be a simple shareable link with a nice GitHub account overview 🙂 

**Q: How does it work?**
> As you may have noticed, render is really similar to SVG renders. It's because metric's [json output feature](https://github.com/lowlighter/metrics/tree/master/source/plugins/core#-convert-output-to-pngjpeg-or-json) is used to render metrics through Vue.js instead of EJS

**Q: Will it support more plugins or include more metrics?**
> Plugins in *GitHub* categorie may be considered in next releases, though some of them may not be because they're either too costly in terms of API requests or in CPU resources, as *Insights* is intended to work mostly on web instances like [metrics.lecoq.io](https://metrics.lecoq.io)

**Q: Will it support additional templates or configuration options?**
> There's no plan to support any options or templates. The aim is to make it very simple and straightforward

**Q: Are organizations and repositories supported?**
> Organizations are supported, tough **isocalendar** plugin is not supported yet. Here's an example for [GitHub organization](https://metrics.lecoq.io/about/github)
>
> Repositories are not supported for now

</details>

I'm not sure whether there's a lot of activity on this repo besides myself (hope I'm not alone 😅), but I'm looking for some feedback, suggestions, ideas and bugs reports on this thread for this new feature 🙂 !  

Since [metrics.lecoq.io](https://metrics.lecoq.io) has been updated to v3.7-beta, you can test [**Insights**](https://metrics.lecoq.io/about) right now!

As a bonus, can you guess who is the most followed user on GitHub? [Click here to know](https://metrics.lecoq.io/about/torvalds)!

**Comment by Nixinova** [comment]
Clipping error here (mobile)
<img src=https://user-images.githubusercontent.com/42429413/120052320-e07f9c00-c078-11eb-981d-e2b102e79116.png width=200px>

**Reply by lowlighter** [reply]
Thanks, will be tracked in #342 👍
